src/java/org/apache/cassandra/db/ReadCommandVerbHandler.java

```diff
@@ -77,8 +84,15 @@ public void doVerb(Message<ReadCommand> message)
         }
 
         Tracing.trace("Enqueuing response to {}", message.from());
-        Message<ReadResponse> reply = message.responseWith(response);
-        MessagingService.instance().send(reply, message.from());
+
+        Optional<Sensor> readBytesSensor = RequestTracker.instance.get().getSensor(Type.READ_BYTES);
+        Message.Builder<ReadResponse> reply = message.responseWithBuilder(response);
+        readBytesSensor.map(s -> SensorsCustomParams.doubleAsBytes(s.getValue())).ifPresent(bytes -> reply.withCustomParam(SensorsCustomParams.READ_BYTES, bytes));
+
+        Optional<Double> aggregareReadBytes = SensorsRegistry.instance.aggregateSensorValueBy(cfs.getKeyspaceName(), Type.READ_BYTES);
```

Comment:
Yeah I misinterpreted

```
The "global" UCU value for the Writer (that is the UCU value across all requests in the Writer, per given tenant).
```

to mean requests across tables for a given keyspace but as you mentioned since the request belongs to a specific table, we should just propagate the monotonically increasing registry sensor valu

Reply:
Yeah sorry for the misunderstanding.